platform-core-module/src/main/java/com/softicar/platform/core/module/CoreModule.java

```diff
@@ -47,4 +49,20 @@ public static EmfPagePath getParentPagePath() {
 
 		return new EmfPagePath().append(PARENT_FOLDER_TITLE);
 	}
+
+	// TODO Remove with PLAT-1099
+	public static <T extends IEmfTableRow<T, ?>> IEmfPermission<T> getModuleAdministation() {
```

Comment:
Shouldn't those methods have a `Permission` suffix? Current naming appears odd to me.

Reply:
they get removed anyway
also, their terminology is the same as in `EmfDefaultModuleRoles`

Reply:
As discussed, should be renamed to `getAdministationPermission`. Same goes for similar methods.